### PR TITLE
Hide capability config for bearer-only clients

### LINKED ISSUE #1277: Bearer only clients should not have Capability config
Bearer only clients are including the `Capability config` section on the Client settings tab.  This section should not be there and should also be removed from the `Jump to section`.

For regular bearer only clients, it is showing the section and all the settings.  For realm clients in master, it is showing the `Capability config` section header with no fields.

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #1277

### CODE REVIEW

**Comment by xianli123** [comment]
Hi @jonkoops, As for bearer-only client, not only the capability configs have been removed, but also the "Login settings" have been removed. In addition the access settings only keep the "Admin URL".  The client-realm also has some changes than the openid-connect client. We may file some different issues to fix them. WDYT? 

Here are the designs FYI. 
Bearer-only clients - https://marvelapp.com/prototype/5di1ce1/screen/81685256
Bearer-only realm-clients - https://marvelapp.com/prototype/5di1ce1/screen/81685255

**Review by xianli123 — APPROVED** [review]
The capability configs have been removed.

**Comment by jonkoops** [comment]
> As for bearer-only client, not only the capability configs have been removed, but also the "Login settings" have been removed. In addition the access settings only keep the "Admin URL". The client-realm also has some changes than the openid-connect client. We may file some different issues to fix them. WDYT?

That is peculiar, I will take a look next week to see if I can resolve these under this PR.

File: src/components/scroll-form/ScrollForm.tsx [hunk 1]
```diff
@@ -67,7 +67,11 @@ export const ScrollForm: FunctionComponent<ScrollFormProps> = ({
           >
             {sections.map((cat) => (
               // note that JumpLinks currently does not work with spaces in the href
-              <JumpLinksItem key={cat} href={`#${spacesToHyphens(cat)}`}>
+              <JumpLinksItem
+                key={cat}
+                href={`#${spacesToHyphens(cat)}`}
```
Comment (edewit) [inline src/components/scroll-form/ScrollForm.tsx:72]: could do `spacesTohyphens` once instead of two times

**Review by edewit — APPROVED** [review]

**Pull request merged**